funsite/templates/funsite/parts/footer.html

```diff
@@ -65,7 +65,7 @@
 
             <div>
                 <a href="//www.fun-mooc.fr" target="_blank">
-                    <img src="${static.url('funsite/images/logos/funmooc-white.svg')}" height="40" alt="Logo FUN Mooc" />
+                    <img src="${static.url('funsite/images/logos/funmooc-color.png')}" height="60" alt="Logo FUN Mooc" />
```

Comment:
Just look'n feel suggestion.
<img width="800" alt="image" src="https://github.com/openfun/fun-apps/assets/9265241/74cefb58-d963-43a6-99ff-9193d450f77f">

```suggestion
                    <img src="${static.url('funsite/images/logos/funmooc-color.png')}" height="60" alt="Logo FUN Mooc" style="border-radius: 4px" />
```